application/controllers/gnu_encryptor.py

```diff
@@ -0,0 +1,28 @@
+import gnupg
+import json
+
+
+class GNUEncrypter:
+
+    def __init__(self, public_key, passphrase=None, always_trust=True):
+        self.gpg = gnupg.GPG()
+        self.gpg.import_keys(public_key.encode('utf-8'))
+
+    def encrypt(self, payload, recipient):
+        """
+        Encrypts the payload using the recipient values
+
+        :param payload: the value to encrypt
+        :param recipient: who is it for
+        :return: string of encrypted data
+        """
+        payload = 'hello world'
+        enc_data = self.gpg.encrypt(json.dumps(payload).encode('utf-8'), recipient, always_trust=True)
+        if not enc_data.ok:
+            print('ok: ', enc_data.ok)
```

Comment:
These shouldn't be print statements.  You should use a logger for this.  Something like
```python
logger.error("Failed to encrypt with gpg", status=enc_data.status, error=enc_data.stderr, recipient=recipient)
```